qbraid/runtime/enums.py

```diff
@@ -52,6 +52,18 @@ class ExperimentType(Enum):
     OTHER = "other"
 
 
+class Paradigm(Enum):
```

Comment:
Hi @rryoung98 , can you explain a bit why this class was required?

Reply:
Same question